<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MapEditor</class>
 <widget class="QMainWindow" name="MapEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Risk Map Editor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLineEdit" name="filenameLineEdit">
        <property name="text">
         <string>../../../maps/Super/SuperMap.map</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="browsePushButton">
        <property name="toolTip">
         <string>Click to browse our computer to find a map to load into the map editor.</string>
        </property>
        <property name="toolTipDuration">
         <number>3</number>
        </property>
        <property name="text">
         <string>Browse...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="loadPushButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Clicking on this button will load a bitmap with it's associated map file.</string>
        </property>
        <property name="text">
         <string>Load Map</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="clearMapPushButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>This will clear the map, while leaving the bitmap to create your own map on top.</string>
        </property>
        <property name="text">
         <string>Clear Map</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="saveComboBox">
        <property name="toolTip">
         <string>Set format for saving the map file.</string>
        </property>
        <item>
         <property name="text">
          <string extracomment="The format implemented from the website. A variant of CSV.">.map</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string extracomment="XML format automatically derived from the Cereal library">.xml</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="saveMapPushButton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string>Click to save the map file into the build folder.</string>
        </property>
        <property name="text">
         <string>Save Map</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>30</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QToolButton" name="addCountryPushButton">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Add country to the map. &lt;/span&gt;&lt;/p&gt;&lt;p&gt;Click on an empty spot in the map to add a new country. Clicking on an existing country will allow you to change its name.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>+ Country </string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="removeCountryPushButton">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Remove a country from the map.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;If there are no more countries left in the continent to which it belonged, the continent will also be removed from the map.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>- Country </string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="addNeighbourPushButton">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Add Link between to existing countries. &lt;/span&gt;&lt;/p&gt;&lt;p&gt;Click on two countries you would like to connect.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>+ Link       </string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="removeNeighbourPushButton">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Remove a Link between two countries.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;Select the two countries you want to break apart by clicking on them in any order.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>-  Link      </string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGraphicsView" name="graphicsView">
        <property name="toolTip">
         <string>When no tool is selected, you can click and drag to move countries around.</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuRisk_v_0_0_1">
    <property name="title">
     <string>Risk v.0.0.1</string>
    </property>
   </widget>
   <addaction name="menuRisk_v_0_0_1"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
